nncf/common/utils/backend.py

```diff
@@ -97,5 +82,12 @@ def copy_model(model: TModel) -> TModel:
     """
     model_backend = get_backend(model)
     if model_backend == BackendType.OPENVINO:
+        # TODO(l-bat): Remove after fixing ticket: 100919
         return model.clone()
+    if model_backend == BackendType.TENSORFLOW:
+        # deepcopy and tensorflow.keras.models.clone_model does not works
```

Comment:
What is idea of this comment?

Reply:
So that everyone knows why this method is used instead of `deepcopy` or `clone_model` and does not try to change it.

Reply:
Then let's write TF version.